spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```diff
@@ -200,6 +200,20 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
   }
 
   /**
+   * whether this layer is trainable
+   */
+  var trainable: Boolean = true
```

Comment:
only layers with parameters are trainable, right? the default value should be false